sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConnection/EventHubConnectionLiveTests.cs

```diff
@@ -211,7 +205,7 @@ await using (var connection = new TestConnectionWithTransport(connectionProperti
 
                     Assert.That(partitionProperties, Is.Not.Null, "A set of partition properties should have been returned.");
                     Assert.That(partitionProperties.Id, Is.EqualTo(partition), "The partition identifier should match.");
-                    Assert.That(partitionProperties.EventHubName, Is.EqualTo(connectionProperties.EventHubName).Using((IEqualityComparer<string>)StringComparer.InvariantCultureIgnoreCase), "The Event Hub path should match.");
+                    Assert.That(partitionProperties.EventHubName, Is.EqualTo(scope.EventHubName).Using((IEqualityComparer<string>)StringComparer.InvariantCultureIgnoreCase), "The Event Hub path should match.");
```

Comment:
nifty... I didn't know about the `Using` extension.   Going to file that away for the future.